docs/learn/learn-guides-treasury.md

```diff
@@ -47,6 +47,22 @@ and is
 
 :::
 
+## Creating A Treasury Proposal - Spend Local
+
+:::info "Spend" vs. "Spend Local"
+
+You may notice that the treasury pallet contains two extrinsics - `treasury.spend` and
+`treasury.spend_local`. `treasury.spend_local` (formally called `treasury.spend`) refers to a spend
+of DOT which is locally available, i.e., DOT from the relay chain's treasury. `spend` actually
+allows for a spend to be specified in an asset other than DOT, or even DOT on a parachain like
+[AssetHub](../general/glossary.md#asset-hub).
+
+Unlike `treasuy.spendLocal`, `treasury.spend` is **not** bound by a spend period, and must be
+claimed manually via the `treasury.payout` extrinsic. `treasuy.spendLocal` behavior remains
```

Comment:
Please choose a style, `spend_local` or `spendLocal`, and stick to it. Both are fine, inconsistency is not.